tools/RAiDER/losreader.py

```diff
@@ -685,17 +686,21 @@ def getTopOfAtmosphere(xyz, look_vecs, toaheight, factor=None):
     return pos
 
 
-def get_orbit(orbit_file, ref_time, pad=600): 
+def get_orbit(orbit_file, ref_time, pad):
     '''
     Returns state vectors from an orbit file
     '''
     # First load the state vectors into an isce orbit
     import isce3.ext.isce3 as isce
+    # svs  = get_sv(orbit_file, ref_time, pad)
+    # rows = np.stack([[isce.core.StateVector(row[0]), row[1:4], row[4:7]] for row in svs], axis=-1)
+    # orb  = isce.core.Orbit(rows)
+
```

Comment:
Please do not leave code that you used to debug in the code unless (a) you leave comments as to what you are doing to debug the code and (b) you and others will use this. When having to edit someone else's code and commented out code, the new editor has to think if this is purposefully commented out, when was this commented out and what this commented out code is debugging.